src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANG = 100;
+    private static final int MAX_INDEX = 4;
+    private static String COLOR;
+    private final int randomInd = new Random().nextInt(MAX_INDEX);
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (randomInd) {
+            case 0:
```

Comment:
it's better to change switch-case and replace the indexes with enum constants of shapes to make it more clear what cases can you have